packages/wonder-blocks-banner/src/components/banner.tsx

```diff
@@ -296,7 +291,10 @@ const Banner = (props: Props): React.ReactElement => {
                 />
                 <View style={styles.labelAndButtonsContainer}>
                     <View style={styles.labelContainer}>
-                        <BodyText size="small">{text}</BodyText>
```

Comment:
Using a `StyledDiv` for now since we're unable to override BodyText default styles currently (WB-1978). The font size changes between Classic and TB so we use the component theme tokens (the css variables) instead